test/turn_test.rb

```diff
@@ -12,44 +12,46 @@ def test_the_turn
     assert_instance_of Turn, turn
   end
 
-  # def test_turn_card_question
-  #   card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
-  #   turn = Turn.new("Juneau", card)
-  #
-  #   assert_equal "What is the capital of Alaska?", turn.card
-  # end
-
-  # def test_turn_card_answer
-  #   card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
-  #   turn = Turn.new("Juneau", card)
-  #
-  #   assert_equal "Juneau", turn.card
-  # end
-  #
-  # def test_turn_card_category
-  #   card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
-  #   turn = Turn.new("Juneau", card)
-  #
-  #   assert_equal :Geography, turn.card
-  # end
   def test_turn_guess
     card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
     turn = Turn.new("Juneau", card)
 
     assert_equal "Juneau", turn.guess
   end
 
+  def test_turn_card
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    assert_equal card, turn.card
+  end
+
   def test_turn_correct?
     card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
     turn = Turn.new("Juneau", card)
 
     assert_equal true, turn.correct?
   end
 
+  def test_turn_fail_correct?
+    card = Card.new("Which planet is closest to the sun?", "Mercury", :STEM)
```

Comment:
It can help out people reading your code to be even more explicit with your variable name. For instance, here I would name the turn variable `incorrect_turn` and the one in `test_turn_correct?` `correct_turn`. One of the quotes that's stuck with me these last couple months is from Joel Spolsky: "It's easier to write code than to read it". If you make your code as readable and understandable as possible, people are less likely to try to scratch it and start over and more likely to be happy with you. And who doesn't like happy people?